README.md

```diff
@@ -1,22 +1,60 @@
 # React Event System
 
 ## Objectives
-
 1. Explain how React events differ from browser events
 2. Describe how React standardizes events for compatibility
-3. Describe the trade-offs of the React event system
-
-## Overview
-
-Introduce students to React Synthetic Events, especially the big ones like
-form events, mouse events, and keyboard events.
-
-Highlight event reuse (practical upshot: we need to log out the event, and can't
-inspect it in console without doing so).
-
-Don't worry about going too deep; there are a few lessons to cover all of this
-stuff.
+3. Explain how we use React events in our application
+
+## What's the event system in React?
+React has its own event system with special event handlers called `SyntheticEvent`. The reason for having a specific
+event system instead of using native events is cross-browser compatibility. Some browsers treat events differently, and
+by wrapping these events into a consistent API, React makes our lives a lot easier. It's important to keep in mind that
+they are the _exact same events_, just implemented in a consistent way! That means these events also have methods like
+`preventDefault()`, `stopPropagation()`, and so on.
+
+## How to add event handlers
+Consider the following component:
+
+```js
+class Tickler extends React.Component {
+  constructor() {
+    super();
+    
+    this.greet = this.greet.bind(this);
```

Comment:
I think you meant this to be: `this.tickle = this.tickle.bind(this);`

Interestingly, I am able to get this to run correctly without binding it. I'm not sure why at the moment. But take a look here: https://jsfiddle.net/n378L25z/